src/include/storage/tile_group_header.h

```diff
@@ -58,10 +61,13 @@ class TileGroup;
  *
  *  STATUS:
  *  ===================
- *  TxnID == INITIAL_TXN_ID, BeginTS == MAX_CID, EndTS == MAX_CID --> empty version
+ *  TxnID == INITIAL_TXN_ID, BeginTS == MAX_CID, EndTS == MAX_CID --> empty
+ * version
  *  TxnID != INITIAL_TXN_ID, BeginTS != MAX_CID --> to-be-updated old version
- *  TxnID != INITIAL_TXN_ID, BeginTS == MAX_CID, EndTS == MAX_CID --> to-be-installed new version
- *  TxnID != INITIAL_TXN_ID, BeginTS == MAX_CID, EndTS == INVALID_CID --> to-be-installed deleted version
+ *  TxnID != INITIAL_TXN_ID, BeginTS == MAX_CID, EndTS == MAX_CID -->
+ * to-be-installed new version
+ *  TxnID != INITIAL_TXN_ID, BeginTS == MAX_CID, EndTS == INVALID_CID -->
+ * to-be-installed deleted version
```

Comment:
You should probably undo the formatting changes here for readability reasons. To selectively bypass formatting, demarcate the code segment with `// clang-format off` and `// clang-format on`.